core/src/main/java/org/frankframework/scheduler/job/SendMessageJob.java

```diff
@@ -53,7 +53,7 @@ public void configure() throws ConfigurationException {
 
 		super.configure();
 
-		localSender = SpringUtils.createBean(getApplicationContext(), SendMessageJobSender.class);
```

Comment:
Deze gaat dan niet goed, kunnen we ook op een andere manier oplossen, hier is de class handig/nodig.

Reply:
Ik doe nog even een check op het hele PR of ik ergens anders nog eenzelfde vergissing heb gemaakt.

Reply:
Volgens mij ziet de rest er goed uit.